webapp/utils/utils.jsx

```diff
@@ -330,32 +324,13 @@ export function extractLinks(text) {
     // strip out inline markdown images
     inText = inText.replace(/!\[[^\]]*\]\([^\)]*\)/g, '');
 
-    function replaceFn(autolinker, match) {
-        let link = '';
-        const matchText = match.getMatchedText();
+    links = inText.match(pattern) || [];
```

Comment:
Also, if you get rid of the /g, you can simplify the return to
```
const match = pattern.exec(inText);
if (match) {
    return match[0].trim();
}

return '';
```
so it doesn't need to use map at all